demos/sev-demo/README.md

```diff
@@ -0,0 +1,206 @@
+```
+# Enable guest attestation
+# (default: false)
+guest_attestation=true
+#
+# Guest owner proxy that handles remote attestation
+guest_attestation_proxy="localhost:50051"
+#
+# Preattestation keyset ID for injected secrets
+guest_attestation_keyset="KEYSET-1"
+```
+Here, we are specifying a local KBS. You can also run the KBS remotely. For this demo we specify the `keyset` in the configuration file. In the future this will be overridden via a Kubernetes annotation. The keyset parameter will be described more below. For now, you can just leave it set to `KEYSET-1`. 
+
+Finally, switch from `virtio-fs` to `virtio-9p`. The former is not currently compatible with SEV. 
+```
+# Shared file system type:
+#   - virtio-fs (default)
+#   - virtio-9p
+shared_fs = "virtio-9p"
+```
+
+## KBS 
+
+You can use the `offline_sev_kbc` manually by injecting a specially formatted JSON file, but you will probably want to use our [key broker server](https://github.com/confidential-containers-demo/scripts/tree/main/guest-owner-proxy) instead. 
+Note that the SEV KBS is often referred to as a Guest Owner Proxy (GOP). This was our original terminology. 
+First, install `gop-client.py` on the worker node by placing the file at `/opt/sev/guest-owner-proxy/gop-client.py`. You should check the `gop-client.py` file and make sure that the `hw_api_major`, `hw_api_minor`, and `hw_build_id` match the parameters reported by the AMD-SP. 
+You can verify these parameters using [SEV-Tool](https://github.com/AMDESE/sev-tool). These parameters will be set automatically in the future and the functionality of `gop-client.py` may be added to the Kata runtime.
+
+You can run `gop-server.py` wherever you would like, just make sure that `guest_attestation_proxy` is set accordingly.
+
+
+The KBS is configured using two files. First, `keys.json` stores the keys that the KBS can send to a guest. Generating these keys is described below. There is also a file called `keysets.json` which defines groups of keys and the policies that correspond to their release. Since the `offline_sev_kbc` is an offline KBC, keys cannot be request individually at runtime. Instead, all the keys that are needed for the duration of the guest need to be provided at boot. Thus, we use a keyset to group keys into one bundle that is injected. 
+
+We can step through the keysets file.
+```
+{
+   "KEYSET-1":{
+```
+Currently we are hardcoding our keyset in the Kata configuration file (see above) to match `KEYSET-1` specified here. 
+```
+      "allowed_digests":[
+         "a11cc9f32b3e64a0846b5eadf773e5e4ee7b4187d984b7a72a3b4fc99e34374c",
+         "dcf332c8abb68cc534d7f1e523ffe9e072308e0d92dc98e3884b313caaea379b"
+      ],
+```
+Firmware digests that are allowed for this keyset. You can calculate a firmware digest using the `calculate_hash.py` tool. Be sure to specify the correct, kernel, initrd, firmware, and kernel parameters. These will be included in the hash (this is an extension of the original SEV behavior). 
```

Comment:
Should we include `calculate_hash.py` (for that matter, all of the `gop-*` scripts) in this sev-demo directory?

Reply:
No, better to have them all in a separate `sev-demo-files` repo (or something similar)

Reply:
I am on the fence about this. For now we might as well have them separate, but maybe at some point we can move them into some confidential-containers repo.